scene/resources/curve.h

```diff
@@ -181,7 +182,16 @@ class Curve2D : public Resource {
 
 	real_t bake_interval = 5.0;
 
+	struct Interval {
+		int idx;
+		real_t frac;
+	};
```

Comment:
It probably should have been pointed out at the time of Curve3D, but it could have been written:
```cpp
typedef Pair<int, real_t> Interval;
```
Well, it is not a blocker.